### ISSUE TITLE
Final review by Wenchang Yang

### ISSUE THREAD

**Issue by SSTXNEO** [terterfred/MergerArbitrage#13]
This project is trying to predict whether or not a company's acquisition case will succeed or fail, which according to the report, reduced to a classification problem. This idea is convincing and clearly founded. Potentially, the result of this project could help with the decision making process of a acquisition case. 

Looking into the details, however, some parts of the report could be further improved. 
First, In the report's explanatory data analysis part, authors put many details into explaining the sources of data sets and missing value problems of the data sets. While the model selection and model interpretation parts did not provide throughout explanation and mathematical interpretation on why they chose such models (Logistic regression, random forest etc.,) to perform analysis and how the models' results reflected the nuances between features. All four models' results were given interpretation of 3-4 sentences without detailed explanation, which made the report a little hard to follow.

Second, the practice of putting model result in the end of the report brought difficulties for readers. I would suggest authors guide readers through the report with both mathematical explanation and text interpretation instead of letting readers jump around the report.

To summarize, this project is built on a enlightening idea and a convincing stepping-point. If authors could give more throughout explanation on models' results, this project would be helpful to acquisition's decision making process.
